Dockerfile

```diff
@@ -1,8 +1,16 @@
-FROM praekeltfoundation/django-bootstrap:py2.7-jessie
-RUN apt-get-install.sh git nodejs npm \
+FROM praekeltfoundation/django-bootstrap:py3.6-jessie
+RUN apt-get-install.sh git \
     redis-server supervisor libpq-dev gcc && \
     ln -s /usr/bin/nodejs /usr/bin/node
 
+#Upgrade Nodejs to v4.9 as the current version v4.6 doesn't appear to have support for object.assign 
+#Check https://node.green for compatibility
+RUN apt-get update
+RUN apt-get install curl -y
+RUN curl -sL https://deb.nodesource.com/setup_4.x | bash
+RUN apt-get install nodejs -y
```

Comment:
I think we probably want to use the `apt-get-install.sh` script here, as that cleans up package indexes and does an apt-get update for us. We can probably add these to the above command to have it only result in a single docker layer being added.